cfme/tests/automate/test_service_automate.py

```diff
@@ -397,4 +400,8 @@ def test_import_dialog_file_without_selecting_file():
             1. Flash message: "At least 1 item must be selected for export"
             2. Error flash message should not appear
     """
-    pass
+    import_export = DialogImportExport(appliance)
+    view = navigate_to(import_export, "DialogImportExport")
+    view.export.click()
+    view.flash.assert_message("At least 1 item must be selected for export")
+    dialog.update({'label': fauxfactory.gen_alphanumeric()})
```

Comment:
Ah thanks @sshveta, yeah @ganeshhubale you need to check the flash message after the update too

Reply:
We don't need to check flash message after dialog update. As we already have 
```view.flash.assert_no_error()``` in **update** method. On update, it gives ```AssertionError: assert_no_error: error: At least 1 item must be selected for export```

Reply:
Ah, thanks for the clarification!